Back/controller/controller_marketplaces.py

```diff
@@ -1,6 +1,6 @@
 from Back.models.model_marketplaces import Marketplace
 from Back.models.model_logs import Log
-from Back.controller.controller_logs import save_l
+#from Back.controller.controller_logs import save_l
```

Comment:
Se puder retirar os códigos comentados, o abreu comentou comigo que não é legal deixar códigos comentados. Caso precise usar novamente no futuro, é melhor puxar pelo git.

Reply:
Isso para os demais comentários também

Reply:
Deixei comentado porque e parte do código do Marcos, ele ajustou isso na branch dele. Mas já subo um commit sem!

Reply:
Eae Lucas Torres, tudo bem?
Esses comentários ele fez para não dar erro de importação nos arquivos que eu estava desenvolvendo na minha PR, quando for feito o merge da minha isso será eliminado na resolução do conflito. Os arquivos de Marketplaces e Sellers ficaram em minha responsabilidade.

Reply:
Ah então beleza